### ISSUE TITLE
symmetry expansion broken links for images

### ISSUE THREAD

**Issue by pconesa** [scipion-em/scipion-em-relion#372]
hi, this has happened to one of our users. He moved the project to another machine and images where gone.

The reason is here:
![image](https://user-images.githubusercontent.com/785633/212132810-af60f613-641e-43f1-abb7-27c4669f8449.png)

fileName points to the "input" folder, which is an absolute link to another runs folder

![image](https://user-images.githubusercontent.com/785633/212133050-7c1faa58-55d1-46bd-8d8f-7250125c36d0.png)

Probably making the link relative will fix it or in the sqlite save the actual path removing "input" folder from the equation.

**Comment by delarosatrevin** [comment]
Yes, if it is a link to another runs location a relative link might be better

**Comment by azazellochg** [comment]
https://www3.mrc-lmb.cam.ac.uk/relion/index.php/FAQs#What_is_the_right_way_to_import_files_outside_the_project_directory.3F
"Symbolic links to somewhere outside the project directory must be by an absolute path. "
